src/plugins/intel_cpu/src/nodes/common/cpu_convert.cpp

```diff
@@ -545,8 +579,22 @@ void cpu_convert(const void *srcPtr,
         } else {
             cpu_memcpy(dstPtr, srcPtr, size * dstPrc.size());
         }
+    } else if (srcPrc == Precision::BIN) {
```

Comment:
May I ask you to extend implementation to cover dstPrc=BIN? We are trying to preserve consistence of Convert operation: for all other data types conversions is supported vice versa.

Reply:
I didn't cover this case, because I couldn't enable the tests to check it. It is possible would be better to do it in a separate task when the BIN format is supported.

Reply:
Got it. Thanks. Lets consider as a separate task then.